lesson_01/chigazograham/README.html

```diff
@@ -0,0 +1,57 @@
+
+
+<h3 style="color: rgb(211, 215, 207); font-family: sans-serif;">Learning Style:</h2>
+<ul>
+    <li style="color: rgb(211, 215, 207); font-family: sans-serif">I am a <b>multimodal learner</b> (<i>Uses multiple modes or methodologies to learn a concept</i>)</li>
+    <li style="color: rgb(211, 215, 207); font-family: sans-serif">My strongest learning styles are <b>kinesthetic</b> (<i>Learning that involves physical activity</i>) and <b>visual learning</b> (<i>Learning through information presented in a visual format</i>)</li>
+    <li style="color: rgb(211, 215, 207); font-family: sans-serif">My weakest learing style is <b>auditory learning</b> (<i>Learning by hearing and listening</i>)</li>
+</ul>
+
+
+
+<h3 style="color: rgb(211, 215, 207); font-family: sans-serif;">Work Hours & Preferences:</h2>
+<ul>
+    <li style="color: rgb(211, 215, 207); font-family: sans-serif">I am <strong>NOT</strong> a morning person, but I am available to work at any time <strong>as long as the meeting time is discussed and agreed upon at least 24 hours before said meeting.</strong></li>
+    <li style="color: rgb(211, 215, 207); font-family: sans-serif">I prefer chats due to ease of use but I am available for in-person meetings if properly scheduled.</li>
+    <li style="color: rgb(211, 215, 207); font-family: sans-serif">In order the most effective method of communication to me would be text message, then email, and finally LinkedIn.</li>
+</ul>
+
+
+<h3 style="color: rgb(211, 215, 207);font-family: sans-serif;">Feedback:</h2>
+<ul>
+<li style="color: rgb(211, 215, 207); font-family: sans-serif"><strong>Feedback, criticism, and advice  is always welcomed!</strong> I prefer to give and receive feedback via email</li>
+<li style="color: rgb(211, 215, 207); font-family: sans-serif">During <strong>weekdays</strong> I try to check my email <strong>every hour from 9 am to 9 pm</strong> unless I am preoccupied with more important work.</li>
+<li style="color: rgb(211, 215, 207); font-family: sans-serif">On <strong>weekends</strong> I usually check my emails three times a day from <strong>9 am to 3 pm, once in the morning, at noon, and in the afternoon</strong>.</li>
+</ul>
+
+<h3 style="color: rgb(211, 215, 207); font-family: sans-serif;">Facts That Aren't Really Fun:</h2>
+<ol>
+<li style="color: rgb(211, 215, 207); font-family: sans-serif"><b>My hometown:</b> Wilmington, Delaware</li>
+<li style="color: rgb(211, 215, 207); font-family: sans-serif"><b>My birthday:</b> September 19th (<i>I just turned 20!</i>)</li>
+<li style="color: rgb(211, 215, 207); font-family: sans-serif"><b>My favorite baked good:</b> Croissants</li>
+<li style="color: rgb(211, 215, 207); font-family: sans-serif"><b>My coffee order:</b> French Vanilla Hazelnut Iced Coffee Regular Cream & Sugar</li>
+<br>
+
+
+<img src="images/GSF_Comp.jpeg" alt="Business pitch competition winners podium" style="color: rgb(211, 215, 207);">
```

Comment:
Images still broken here and below. If not fixed by end of day, I'll just approve so we can get this submitted.